lib/promiseFtp.coffee

```diff
@@ -106,7 +105,7 @@ class PromiseFtp
       
     # methods listed in otherPrototypeMethods, which don't get a wrapper
     
-    @connect = (options) -> Promise.try () ->
+    @connect = (options) -> Promise.resolve().then () ->
```

Comment:
Also the loss of `.finally` without bluebird really sucks from a connection maintenance standpoint.